src/components/Button/index.js

```diff
@@ -1 +1,2 @@
-export { default } from "./Button.tsx";
+export { Button } from "./Button.tsx";
```

Comment:
Please revert this line. The Button component is exported using:
`export default Button;`
and for that reason the export here could be:
1. export { default } from "./Button.tsx";
or
2. export { default as Button } from "./Button.tsx";